languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -28745,4 +28607,609 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
         </rulegroup>
     </category>
 
+   <!-- ====================================================================== -->
+   <!-- Hilfestellung für Kommasetzung -->
+   <!-- ====================================================================== -->
+   <category id="HILFESTELLUNG_KOMMASETZUNG" name="Hilfestellung für Kommasetzung">
+		
+	    <rulegroup id="KOMMA_INFINITIVGRUPPEN" name="Komma vor Infinitivgruppen mit 'als', 'anstatt', 'außer', 'ohne', 'statt', 'um'">
+		    <url>http://www.duden.de/sprachwissen/rechtschreibregeln/komma</url>
+		    <rule>
+				<antipattern> <!-- Ausnahme: verbale Klammer - testen!  -->
```

Comment:
Wir benutzen in dieser Datei sonst überall vier Leerzeichen zur Einrückung. Bitte nicht Tabulatoren und Leerzeichen mischen, das gibt nur Chaos. Vielleicht hilft das hier weiter: http://codebeautify.org/xmlviewer